src/content/contribute/writing-a-loader.md

```diff
@@ -342,6 +345,7 @@ export default (fixture, options = {}) => {
   });
 
   compiler.outputFileSystem = createFsFromVolume(new Volume());
+  compiler.outputFileSystem.join = path.join.bind(path);
```

Comment:
We need this line, I've copied it from https://github.com/webpack/webpack-dev-middleware/blob/094693854d9461b7dfb6cc371a16260e3c02a36b/src/utils/setupOutputFileSystem.js#L30